components/LandingPage.js

```diff
@@ -1,22 +1,26 @@
 import React from "react";
-import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+import { View, Text, StyleSheet, TouchableOpacity, ImageBackground } from "react-native";
 import { Header } from 'react-native-elements';
 import normalize from 'react-native-normalize';
 
 const LandingPage = ({ navigation }) => {
   return (
     <View style={styles.container}>
       <Header
-        containerStyle={{ backgroundColor: 'black', justifyContent: 'space-around', paddingBottom: 10}}
+        containerStyle={{ backgroundColor: '#483D8B', justifyContent: 'space-around', paddingBottom: 10}}
```

Comment:
Dang, I added a bunch of comments to the other PR. Oh well. Yeah, we can pull these styles out into their own StyleSheet. Very low priority, just making a mention of it.